diff --git a/dist/index.cjs b/dist/index.cjs
index 61d52cbb39503c92fafb84e9207ba52d317aed97..8c2b12fb14a449512ae2319437154394796c37d2 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -1871,6 +1871,9 @@ function MainPlugin(ctx, api) {
       };
     },
     async configResolved(config) {
+      if (config.build.ssr) {
+        return;
+      }
       ctx.useImportRegister = false;
       ctx.viteConfig = config;
       ctx.userOptions?.integration?.configureOptions?.(config, ctx.userOptions);
diff --git a/dist/index.js b/dist/index.js
index 84bf280f220d12313d0523b2160f7f6eb64fd81a..e1557ff7634266f4941d3b5a3120039e2d7a97ac 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -984,6 +984,9 @@ function MainPlugin(ctx, api) {
       };
     },
     async configResolved(config) {
+      if (config.build.ssr) {
+        return;
+      }
       ctx.useImportRegister = false;
       ctx.viteConfig = config;
       ctx.userOptions?.integration?.configureOptions?.(config, ctx.userOptions);
